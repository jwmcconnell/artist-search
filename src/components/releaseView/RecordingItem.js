import React from 'react';
import PropTypes from 'prop-types';

const RecordingItem = ({ recording }) => {
  return (
    <li>
      <h3>{recording.title}</h3>
    </li>
  );
};

RecordingItem.propTypes = {
  recording: PropTypes.object.isRequired
};

export default RecordingItem;
