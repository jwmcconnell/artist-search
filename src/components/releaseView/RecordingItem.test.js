import React from 'react';
import { shallow } from 'enzyme';
import RecordingItem from './RecordingItem';

describe('RecordingItem component', () => {
  it('renders RecordingItem', () => {
    const wrapper = shallow(<RecordingItem recording={{}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
