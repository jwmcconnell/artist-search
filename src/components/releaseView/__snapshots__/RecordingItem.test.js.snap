// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecordingItem component renders RecordingItem 1`] = `
<li>
  <h3 />
</li>
`;
